modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -1944,28 +1942,24 @@ describe('V2 Wallet:', function () {
 
       messageSigningCoins.map((coinName) => {
         tssEthWallet = new Wallet(bitgo, bitgo.coin(coinName), ethWalletData);
+
         it('should sign message', async function () {
+          nock(bgUrl)
+            .get(`/api/v2/wallet/${tssEthWallet.id()}/txrequests?txRequestIds=${txRequestForMessageSigning.txRequestId}&latest=true`)
+            .reply(200, { txRequests: [txRequestForMessageSigning] });
 
           const signMessage = await tssEthWallet.signMessage({
             reqId,
-            messagePrebuild: { message: 'test', txRequestId: 'id' },
+            messagePrebuild: { message: 'test', txRequestId: txRequestForMessageSigning.txRequestId },
```

Comment:
should there be a test for when there is a `txRequestId` provided and one for when there is not a `txRequestId` provided? It seems like that is the primary goal of this PR

Reply:
The one for no request Id provided would have to get it from WP, which would be an integration test at that point. Unless we want to return a mock one. I didn't see the usefulness of returning a mock id.

Reply:
I think adding a mocked test would check if/else logic in `signMessageTss()` - am I thinking about this wrong?